test/index.ts

```diff
@@ -36,6 +38,16 @@ const options: { [key: string]: string | boolean | number } = {
 //
 // See https://github.com/mochajs/mocha/wiki/Using-mocha-programmatically#set-options for all available options
 
+// For local testing, dotenv will load in the local .env file.  Change the path variable to the absolute path of the .env file.
+const localEnvVars: dotenv.DotenvConfigOutput | undefined = dotenv.config({ path: 'C:/Users/naturins/Desktop/Code/vscode-azureappservice/.env' });
```

Comment:
Soo we definitely don't need to have your 'naturins' path checked in - but I also don't understand why all of this ".env" stuff is necessary anyways. Can't you just set environment variables on your system?

Reply:
Yes, we could, but it's harder to explain and to set up.  Plus I don't know if setting environment variables are the same on every platform, but `dotenv` should work for everything.  For other extensions that don't include `dotenv` already, it can just be a devDependency.